### PR TITLE
feat(getCssVarValue): utility function

### PR DESCRIPTION
**Pull request by atabel**
WEB-1518

### CODE REVIEW

File: src/utils/dom.tsx [hunk 1]
```diff
@@ -121,3 +121,32 @@ export const listenResize = (element: Element, handler: ResizeListener): (() =>
...
+
+const parseVarCall = (cssVar: string) => {
+    const [varName, fallback] = cssVar
+        .replace(/^var\(/, '')
+        .replace(/\)$/, '')
+        .split(/\s*,\s*(.*)/);
+    return {varName, fallback};
+};
+
+const isCssVar = (cssVar: string) => cssVar.startsWith('var(') || cssVar.startsWith('--');
```
Comment (marcoskolodny) [inline src/utils/dom.tsx:133]: Should we consider handling cases in which this returns true, but it's actually not a valid css var? (something like `(varrrr` for example).

Comment (atabel) [reply src/utils/dom.tsx:133]: I think this is good enough. But I don't understand your example, `(varrrr` doesn't start with `var(`

**Review by pladaria — APPROVED** [review]

File: src/utils/dom.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/utils/dom.tsx:133]: Sorry, I meant something line `var(((`.

Comment (atabel) [reply src/utils/dom.tsx:133]: I think it doesn't make sense to cover that case, this function will probably be used with mistica skinVars in most of the cases, so we won't have these weird cases

**Pull request merged**